test/e2e/parallel/retarget_test.go

```diff
@@ -20,20 +20,18 @@ func TestRetargetUserByChangingSpaceTargetClusterWhenSpaceIsNotShared(t *testing
 	member1Await := awaitilities.Member1()
 	member2Await := awaitilities.Member2()
 
-	userSignup, mur := NewSignupRequest(awaitilities).
+	user := NewSignupRequest(awaitilities).
 		ManuallyApprove().
 		TargetCluster(member1Await).
 		EnsureMUR().
 		RequireConditions(ConditionSet(Default(), ApprovedByAdmin())...).
-		Execute(t).Resources()
-
-	space, err := hostAwait.WaitForSpace(t, userSignup.Status.CompliantUsername,
-		UntilSpaceHasStatusTargetCluster(member1Await.ClusterName),
-		UntilSpaceHasAnyTierNameSet())
-	require.NoError(t, err)
+		Execute(t)
+	userSignup := user.UserSignup
+	mur := user.MUR
+	space := user.Space
```

Comment:
the same - the vars are used only once, you can reference them through the `user`